wikimedia/wikipedia-ios
# Add back in beta cluster environment as a staging option

**Phabricator:** https://phabricator.wikimedia.org/T288297

### Notes
This restores the previous beta cluster environment as an environment to toggle in `Configuration.swift` staging options. I still enjoy only having two schemes for alternate environments, one for Staging and another for Local, so this option sits up under the Staging scheme since in my mind it's just another staging environment for us to utilize. I was hesitant to create a new Push Notifications app variant and scheme since this beta cluster could be of use in other testing areas besides push notifications, but I'm open to being convinced otherwise.

### Test Steps
1. Run the Staging scheme
2. Proxy to look at the calls
3. Confirm most calls go through a `{lang}.wikipedia.beta.wmflabs.org` subdomain. Especially for login (you will need to make [a new account](https://en.wikipedia.beta.wmflabs.org/w/index.php?title=Special:CreateAccount&returnto=Main+Page) on the beta cluster) and areas we plan to deep link to with notifications, like article view, diff views and user talk pages.
